device/transport/mqtt/src/mqtt.ts

```diff
@@ -1020,6 +1020,16 @@ export class Mqtt extends EventEmitter implements DeviceTransport {
       });
     }
   }
+
+  private _ignoreConnectionClosedInErrorCallback(callback: (err?: Error, ...args: any[]) => void): (err?: Error, ...args: any[]) => void {
```

Comment:
>_ignoreConnectionClosedInErrorCallback [](http://example.com/codeflow?start=10&length=38)

This needs to be explained in both places, otherwise it will become a Chesterson’s Fence. (one of my new favorite obscure terms).